src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/activities/ActivitiesService.kt

```diff
@@ -63,7 +63,7 @@ class ActivitiesService(
   }
 
   private fun buildActivityMappingDto(
-    nomisResponse: ActivityResponse,
+    nomisResponse: CreateActivityResponse,
```

Comment:
Updating the noms-api model picked up a few dot renames and additional mandatory properties